src/pages/SweetSpot.vue

```diff
@@ -0,0 +1,597 @@
+				endExpReq = expNextLevel(charLevel.value + 1);
+
+				// Obtain the difference from their starting point to the level up
+				let exp1 = expReq - expReq*startExp.value/100;
+			
+				// Obtain how much they earned since their level up
+				let exp2 = curEnd;
+				//console.log("exp1 :" + exp1);
+				//console.log("exp2 :" + exp2);
+				//console.log("+1 " + expNextLevel(charLevel.value + 1));
+				expFarmed = exp1 + exp2;
+			}
+			else {
+				// Current EXP of player
+				curEnd = expNextLevel(charLevel.value)*endExp.value/100;
+				// If they didn't level up, the EXP required is the one
+				// needed for the original level
+				endExpReq = expReq;
+				expFarmed = curEnd - curStart;
+			}
+
+	
+			hourDiff = hourDiff / 60 / 60 / 1000;
+			let expHr = expFarmed / hourDiff;
+			let remainTime = (endExpReq - curEnd) / expHr;
+
+			/*console.log("hourDiff " + hourDiff);
+			console.log("curStart " + curStart);
+			console.log("curEnd " + curEnd);
+			console.log("ExpFarmed " + expFarmed);
+			console.log("Remaing to farm" + (expReq - curEnd));
+			console.log("exp/Hr " + expHr);*/
+
+			return [remainTime,expHr];
+		}
+
+		const chanceToHit = (monster: Monster): number  => {
+			// Edge Case
+			if(accuracy.value < 2) return 0;
+	
+			let hitChance = ((accuracy.value - monster.accLow)/(monster.accHigh - monster.accLow)*95) + 5;
+			// 5% is minimum required to hit a mob
+			if (hitChance < 5) { return 0 };
+			return Math.min(hitChance,100)/100;
+
+		}
+
+		const avgSwingToHit = (monster: Monster): number => {
+			let hitChance = chanceToHit(monster);
+			if (hitChance == 0) { return 0 };
+			return 1/hitChance;
+		}
+
+		/* 
+		 * The next two functions are technically not the real min/max
+		 * They are used to calculate avgHitToKill.
+		 */
+		const minHitToKill = (monster: Monster): number => {
+			let hit = monster.health / (maxDmg.value*multiplier.value);
+			return Math.ceil(hit);	
+		}
+
+		const maxHitToKill = (monster: Monster): number => {
+			let hit = monster.health / (minDmg.value*multiplier.value);
+			return Math.ceil(hit);	
+		}
+
+		const avgHitToKill = (monster: Monster): number => {
+			let minHit = minHitToKill(monster);
+			let maxHit = maxHitToKill(monster);
+
+			return ((minHit + maxHit)/2);
+		}
+
+		const avgSwingToKill = (monster: Monster): number => {
+			let swingToHit = avgSwingToHit(monster);
+			let hitToKill = avgHitToKill(monster);
+			return swingToHit*hitToKill;
+		}
+
+		const monsterExpMul = (monster: Monster): number => {
+			//console.log(ingameEXPMonst.value);
+			//console.log(monstEXPMul.value);
+			return monster.exp*monstEXPMul.value;
+		}
+
+		const expPerSwing = (monster: Monster): number => {
+			let swingToKill = avgSwingToKill(monster);		
+			let expMul = monsterExpMul(monster);
+			return expMul/swingToKill;
+		}
+		
+		const bestMob = (monster: Monster): number => {
+			// World 3 not ready
+			if(monster.numMobs == 1) return 0;
```

Comment:
(nit, ish): always use `===` instead of `==` when doing equality checking in JS. That makes sure `'1' === 1` return `false`. `'1' == 1` is `true` (JS is weird).